### DISCUSSION TITLE
iOS widget contributions graph colors in dark theme

### DISCUSSION THREAD

**Discussion by dolfbarr** [community/community#30043] [Mobile] [answered]
It seems like iOS widget contribution graph colors in dark theme are not synced with the web version:

Could it be possible to make them in sync with the web version since it makes more sense to have lighter colors represent more contributions on the dark theme? 

iOS widget:
![IMG_4286](https://user-images.githubusercontent.com/15245834/185574670-99f6b0c2-0880-4b38-8f72-43c9ae7179b9.jpeg)

Web version: 
<img width="226" alt="image" src="https://user-images.githubusercontent.com/15245834/185573816-5ca6727b-e19d-473d-b3d4-4c6617289648.png">

**Comment by taki-on** [comment, marked as answer]
Great catch! We will file it as a bug and try to fix it soon! Thank you very much for the report!

**Reply by dolfbarr** [reply]
Great news, thank you!

**Reply by galaxynot8** [reply]
Hi

**Comment by ghost** [comment]
#35533
